README.md

```diff
@@ -4,8 +4,91 @@ Examples of some of the features of Jigsaw released in the Early Access build of
 
 Many of the examples here are directly from the Project Jigsaw: Module System Quick-Start Guide, see [http://openjdk.java.net/projects/jigsaw/quick-start](http://openjdk.java.net/projects/jigsaw/quick-start) but we have also contributions from the Java community.
 
-___
+## Setup (all platforms)
 
+#### Download and install latest version of the JDK 9 EA
```

Comment:
this block has moved to a new file: `setupAndVerifyJDK,md`, your branch might need updating